### ISSUE TITLE
to video or not to video

### ISSUE THREAD

**Issue by VNGDR** [Feodor2/Mypal#62]
Hi Feodor,

I'm new to Github and Mypal, so excuse me if I am misbehaving in any way.

Just installed Mypal on my XP (can't say goodbye to it....) and I must say it works like clockwork. Things that didn't work properly on Firefox 52.9.0 ESR do work properly on Mypal 28.5.2, so I am happy. Performance is al lot better as well. Tried a lot of different chromium and mozilla flavors the last couple of days, and this is the best browser so far.

One issue has been bothering me the last couple of weeks that has not been solved yet; maybe this is a good case to further improve upon Mypal. It is about video playback on the Dutch public TV site NPOstart.nl.

Basically this URL works properly:
https://www.npostart.nl/verborgen-verleden/21-03-2019/VPWON_1302530
....and this one doesn't:
https://www.npostart.nl/verborgen-verleden/20-10-2018/VPWON_1280999

Must be something trivial, but I've not been able to figure it out.
I've tested Mypal for basic integrity playing video's with:
https://www.youtube.com/html5 > checks all the boxes
http://bubblemark.com/silverlight2.html > balls are bouncing happily

So I'm puzzled, and would appreciate more insight to this.

Rgds, and cheers to the quality of Mypal so far.

Bart

**Comment by Feodor2** [comment]
I have Dit **programma kan in deze browser helaas niet worden afgespeeld (22)** on the first link.
Please translate on english.

**Comment by VNGDR** [comment]
It means "This video unfortunately cannot be played in this browser" - which is a very inaccurate message. What it really means is that you are probably missing the Silverlight Plugin. The version I use that works for me is 5.1.50918.0. With that installed the first link plays properly but the second doesn't

**Comment by VNGDR** [comment]
Hmm, I've continued my quest, and I'll be answering my own question a bit, with assumptions (and you know assumption is....)

Anyways, I think this is all due to Widevine / DRM being very picky and restrictive lately. I guess the video that is actually playing is 'faulty' because it doesn't enforce Widevine/DRM, and so it runs. Other content doesn't because I think it needs Widevine - which I guess Mypal doesn't have on board. Tried other browsers on XP with no luck so far, and I think this is due to the recent restrictions in Widevine. If you take a look here: https://www.widevine.com/news you'll see what I mean.

That leaves just one question? Has anybody had any luck getting Widevine protected video running on XP lately? According to the Widevine story 360 Chrome should be able to work since it is Chrome 69, tried it, and yes, it does come with a Widevine stub, and no, I've not been able to revive it. 

And I read for instance Basilisk should be able to do Widevine DRM, but I guess that doesn't apply for the XP-forks? I think it should be possible to get Widevine running on Mozilla / Firefox forks running on XP; fairly recent sources are here:

https://github.com/MoonchildProductions/UXP/tree/master/dom/media/gmp/widevine-adapter

Or maybe I just have to wait a bit for the so called 'Chrome 76 project' that is supposedly underway, probably linked to Advanced Chrome 54 stuff, see:
https://browser.taokaizen.com/forum/viewtopic.php?p=3219
https://msfn.org/board/topic/178380-extreme-explorer-360-chromium-69-general-discussion/page/4/#comments
https://browser.taokaizen.com/forum/viewtopic.php?t=2766

I hope anybody can deny (or confirm :-) ) my assumptions. Any hints, trick, tips or how-to's on getting Widevine DRM up and running on Windows XP today would be greatly appreciated.

Rgds, Bart
